holoviews/__init__.py

```diff
@@ -32,7 +32,10 @@
 
 try:
     import IPython  # noqa (API import)
-    from .ipython import notebook_extension
+    class notebook_extension(param.ParameterizedFunction):
+        def __call__(self, *args, **opts):
+            from .ipython import load_extension
+            load_extension(*args, **opts)
```

Comment:
Do we really need to rename to ``load_extension``? What about a qualified import, ``ipython.notebook_extension(*args, **opts)``. I would rather avoid renaming it in ``.ipython`` if possible.